src/js/args.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+var fluid = require("infusion");
+
+fluid.registerNamespace("fluid.lintAll");
+
+fluid.lintAll.cleanArg = function (argString) {
+    var cleanedArgString = argString.replace(/^['"](.+)['"]$/, "$1");
+    return cleanedArgString;
+};
+
+fluid.lintAll.parseArgs = function (processArgs) {
```

Comment:
Would minimist be able to do most of the work of this function, perhaps with a few simplifications?

Reply:
It definitely simplified the parsing, although it lacks any kind of validation.